### PR TITLE
Remove sending 'mnot' pixel when notifications disabled

### PR DESCRIPTION
**Pull request by nalcalag**
Task/Issue URL: https://app.asana.com/0/488551667048375/1206420860078750/f

### Description
Check if notifications permissions have been granted in order to send a pixel for engagement notification shown

### Steps to test this PR

_Pre steps_
- [ ] Change notification schedule to 10-20 SECONDS instead DAYS

_Allow notifications permissions_
- [ ] Fresh install from branch
- [ ] Allow notifications when prompt displays
- [ ] Check in logcat `mnot_s_%` pixels are fired

_Reject notifications permissions_
- [ ] Fresh install from branch
- [ ] Reject notifications permissions when prompt displays
- [ ] Check in logcat `mnot_s_%` pixels are **NOT** fired

### CODE REVIEW

**Comment by nalcalag** [comment]
* `develop`
  * **#4118** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4118?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nalcalag and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: app/src/main/java/com/duckduckgo/app/notification/AndroidNotificationScheduler.kt [hunk 1]
```diff
@@ -48,23 +50,25 @@ class NotificationScheduler(
...
+        }
     }
 
     private suspend fun scheduleUnusedAppNotifications() {
         if (privacyNotification.canShow()) {
             scheduleNotification(
                 OneTimeWorkRequestBuilder<PrivacyNotificationWorker>(),
-                PRIVACY_DELAY_DURATION_IN_DAYS,
-                TimeUnit.DAYS,
+                5L,
```
Comment (marcosholgado) [inline app/src/main/java/com/duckduckgo/app/notification/AndroidNotificationScheduler.kt:62]: This need to be rolledback

File: app/src/main/java/com/duckduckgo/app/notification/AndroidNotificationScheduler.kt [hunk 2]
```diff
@@ -48,23 +50,25 @@ class NotificationScheduler(
...
+                TimeUnit.SECONDS,
                 UNUSED_APP_WORK_REQUEST_TAG,
             )
         }
         if (clearDataNotification.canShow()) {
             scheduleNotification(
                 OneTimeWorkRequestBuilder<ClearDataNotificationWorker>(),
-                CLEAR_DATA_DELAY_DURATION_IN_DAYS,
-                TimeUnit.DAYS,
+                8L,
```
Comment (marcosholgado) [inline app/src/main/java/com/duckduckgo/app/notification/AndroidNotificationScheduler.kt:70]: Samesies

**Review by marcosholgado — CHANGES_REQUESTED** [review]
Need to rollback the `scheduleUnusedAppNotifications()` changes

**Review by marcosholgado — APPROVED** [review]
LGTM

**Pull request merged**